docs/api/textarea.md

```diff
@@ -22,314 +17,61 @@ import EncapsulationPill from '@components/page/api/EncapsulationPill';
 
 <EncapsulationPill type="scoped" />
 
-<h2 className="table-of-contents__title">Contents</h2>
-
-<TOCInline
-  toc={toc}
-  maxHeadingLevel={2}
-/>
-
-
-
 The textarea component is used for multi-line text input. A native textarea element is rendered inside of the component. The user experience and interactivity of the textarea component is improved by having control over the native textarea.
 
 Unlike the native textarea element, the Ionic textarea does not support loading its value from the inner content. The textarea value should be set in the `value` attribute.
 
 The textarea component accepts the [native textarea attributes](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/textarea) in addition to the Ionic properties.
 
-## Interfaces
-
-### TextareaChangeEventDetail
-
-```typescript
-interface TextareaChangeEventDetail {
-  value?: string | null;
-}
-```
-
-### TextareaCustomEvent
+## Basic Usage
 
-While not required, this interface can be used in place of the `CustomEvent` interface for stronger typing with Ionic events emitted from this component.
+import BasicPlayground from '@site/static/usage/textarea/basic/index.md';
 
-```typescript
-interface TextareaCustomEvent extends CustomEvent {
-  detail: TextareaChangeEventDetail;
-  target: HTMLIonTextareaElement;
-}
-```
+<BasicPlayground />
```

Comment:
I don't want to sound biased since I worked on input 😆 so if you disagree let me know, but should we add more samples of textarea with a placeholder, readonly, etc like this:

https://ionic-docs-git-fw-1287-ionic1.vercel.app/docs/api/input#basic-usage

Reply:
I added readonly and disabled examples 👍 The others seem a bit trivial to me, and since textarea takes up more room than input, I'd rather restrict the examples a little more.